isodatetime/parser_spec.py

```diff
@@ -171,69 +171,69 @@
 _DATE_TRANSLATE_INFO = [
     (r"\+(?=X)", "(?P<year_sign>[-+])",
      "%(year_sign)s", "year_sign"),
-    ("CC", r"(?P<century>\d\d)",
+    (r"CC", r"(?P<century>[0-9][0-9])",
      "%(century)02d", "century"),
-    ("YY", r"(?P<year_of_century>\d\d)",
+    (r"YY", r"(?P<year_of_century>[0-9][0-9])",
      "%(year_of_century)02d", "year_of_century"),
-    ("MM", r"(?P<month_of_year>\d\d)",
+    (r"MM", r"(?P<month_of_year>[0-9][0-9])",
      "%(month_of_year)02d", "month_of_year"),
-    ("DDD", r"(?P<day_of_year>\d\d\d)",
+    (r"DDD", r"(?P<day_of_year>[0-9][0-9][0-9])",
      "%(day_of_year)03d", "day_of_year"),
-    ("DD", r"(?P<day_of_month>\d\d)",
+    (r"DD", r"(?P<day_of_month>[0-9][0-9])",
      "%(day_of_month)02d", "day_of_month"),
-    ("Www", r"W(?P<week_of_year>\d\d)",
+    (r"Www", r"W(?P<week_of_year>[0-9][0-9])",
      "W%(week_of_year)02d", "week_of_year"),
-    ("D", r"(?P<day_of_week>\d)",
+    (r"D", r"(?P<day_of_week>[0-9])",
      "%(day_of_week)01d", "day_of_week"),
-    ("z", r"(?P<year_of_decade>\d)",
+    (r"z", r"(?P<year_of_decade>[0-9])",
      "%(year_of_decade)01d", "year_of_decade"),
-    ("^---", "(?P<truncated>---)",
+    (r"^---", "(?P<truncated>---)",
```

Comment:
Should `r"..."` the second item here, for consistent style.

Reply:
Good spot! Fixed!